make/containers.mk

```diff
@@ -59,6 +59,7 @@ $(BINDIR)/containers/cert-manager-controller-linux-amd64.tar $(BINDIR)/container
 	@$(eval BASE := BASE_IMAGE_controller-linux-$*)
 	$(CTR) build --quiet \
 		-f hack/containers/Containerfile.controller \
+		--load \
```

Comment:
could you quickly explain here why we load the built image? does this give use any performance benefits?